### PR TITLE
feat: Order articles by updated_at 

### LINKED ISSUE #6312: Help center articles should be shown by last_edited first.
**Is your enhancement request related to a problem? Please describe.**
Help center articles currently are displayed in an in order - First created article, instead it should be either sorted by Last edited or Last created. 

Screenshot2023-01-20 at 13 18 32@2x

**Describe the solution you'd like**
Sort them by last created or last edited, or have a sort option on the table header to do this.

### PR DESCRIPTION
**Pull request by imskr**
# Pull Request Template

## Description

Please include a summary of the change and issue(s) fixed. Also, mention relevant motivation, context, and any dependencies that this change requires.
Fixes #6312 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/controllers/api/v1/accounts/articles_controller.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ def index
     @portal_articles = @portal.articles
     @all_articles = @portal_articles.search(list_params)
     @articles_count = @all_articles.count
-    @articles = @all_articles.page(@current_page)
+    @articles = @all_articles.page(@current_page).order_by_updated_at
```
Comment (scmmishra) [inline app/controllers/api/v1/accounts/articles_controller.rb:11]: This should be sorted before the pagination right? CC: @tejaswinichile

Comment (imskr) [reply app/controllers/api/v1/accounts/articles_controller.rb:11]: Thanks for the suggestion, i believe doing above won't hurt but yeah happy to make the changes as required CC @tejaswinichile

Comment (imskr) [reply app/controllers/api/v1/accounts/articles_controller.rb]: I have made the changes. Sorted before pagination.

**Review by nithindavid — APPROVED** [review]
Tested locally and articles were sorted by updated_at.

**Pull request merged**